jsonnet/components/kube-state-metrics-custom-resource-state.libsonnet

```diff
@@ -1,23 +1,28 @@
-local vpaMetric(name, help, rawType, statesetScope='') = if name == '' || help == '' || rawType == '' then null else {
-  local type = if rawType == 'StateSet' then 'stateSet' else std.asciiLower(rawType),
-  local resourceType = if std.endsWith(name, 'cpu') then 'cpu' else if std.endsWith(name, 'memory') then 'memory' else '',
+local getCommonLabels(resourceType) =
+  if resourceType != '' then {
+    resource: resourceType,
+    unit: if resourceType == 'cpu' then 'cores'
+          else
+             if resourceType == 'memory' then 'bytes'};
```

Comment:
Would it be better to keep this function within the `vpaMetric` function since IIUC that's the only place this is being used in? All else looks good! 👍🏼